### ISSUE TITLE
Add Error Display To Fiddle

### ISSUE THREAD

**Issue by crutchcorn** [BuilderIO/mitosis#1162]
### I am interested in helping provide a fix!

Yes

### Which generators are impacted?

- [ ] All
- [ ] Angular
- [ ] HTML
- [ ] Qwik
- [ ] React
- [ ] React-Native
- [ ] Solid
- [ ] Stencil
- [ ] Svelte
- [ ] Vue
- [ ] Web components

### Reproduction case

https://mitosis.builder.io/?outputTab=M4ewNglgJkA%3D&code=KYDwDg9gTgLgBAE2AMwIYFcA29noHYDGMAlhHnALICeAwhALaR7B4wAUYUEYAzgJRwA3gCg4cKMBjoo5NqLFwAPAmIA3cSgC8gztx4A6GMB4wAvgD4AKsZiKA9CtXn5fANzDTQA%3D

### Expected Behaviour

Code breaks from time-to-time - it happens. However, when it does, we should more eagerly tell the user of the Fiddle when and why there is an error in the compiler so that they're able to debug their input easier (and figure out if it's their fault or a bug in Mitosis)

### Actual Behaviour

Rather than an error, it just displays no content or the contents of the last tab

**Comment by samijaber** [comment]
Agreed! We should improve the error handling of code generation, and maybe output a banner or modal with the error that occurred.

https://github.com/BuilderIO/mitosis/blob/afdb648f30214e9bc1a3262096628831ee3616d5/packages/fiddle/src/components/Fiddle/Fiddle.tsx#L338-L357

This is where we call the generator, and we currently only log the error to the console.
